massa-node/base_config/config.toml

```diff
@@ -210,10 +210,10 @@
     # Nb max in connections that we accept
     max_in_connections = 100
     # Peer default category limits
-    default_category_info = { target_out_connections = 10, max_in_connections_per_ip = 2, max_in_connections_pre_handshake = 70, max_in_connections_post_handshake = 15}
+    default_category_info = { target_out_connections = 10, max_in_connections_per_ip = 2, max_in_connections = 15}
```

Comment:
It would be nice to take the boolean about the possibility to have local address to connect to us directly in the config there.